avocado/core/loader.py

```diff
@@ -795,4 +797,34 @@ def get_decorator_mapping():
         return {test.ExternalRunnerTest: output.TERM_SUPPORT.healthy_str}
 
 
+class RemoteLoader(TestLoader):
+
+    """
+    Remote-runner loader class
+    """
+    name = 'remote'
+
+    def __init__(self, args, extra_params):
+        super(RemoteLoader, self).__init__(args, extra_params)
+        loaders = getattr(args, 'loaders', None)
+        if loaders is not None and 'remote' in loaders:
+            self.remote = True
+        else:
+            self.remote = False
+
+    def discover(self, url, which_tests=DEFAULT):
+        if self.remote:
+            return [(test.UnknownTest, {'name': url})]
+        else:
+            return []
+
+    @staticmethod
+    def get_type_label_mapping():
+        return {test.UnknownTest: 'REMOTE'}
```

Comment:
Also if you change the name to `DummyLoader`, I'd consider using `DummyTest` as test type :wink: